src/all_shaders.zig

```diff
@@ -37,7 +39,7 @@ pub const ShaderProgram = struct {
     pub fn attribLocation(sp: ShaderProgram, name: [*]const u8) c.GLint {
         const id = c.glGetAttribLocation(sp.program_id, name);
         if (id == -1) {
-            _ = c.printf(c"invalid attrib: %s\n", name);
```

Comment:
I think you can further reduce this by using `std.debug.panic` and removing the abort() call:

```zig
std.debug.panic("invalid attrib: {}\n", name);
```

Reply:
As you say